docs/building-images.md

```diff
@@ -67,6 +67,22 @@ operatingSystem:
   * `enable` - Optional; List of systemd services to enable.
   * `disable` - Optional; List of systemd services to disable.
 
+## SUSE Manager (SUMA)
+
+Automatic SUSE Manager registration can be configured for the image. The following items must be defined in the
+configuration file under the `suma` section:
+
+* `host` - This is the FQDN of the SUSE Manager host that the host needs to register against.
+* `activation_key` - This is the activation key that the node uses to register with.
+* `get_ssl` - This specifies whether EIB should download and install the SUMA SSL Certificate (default: false)
```

Comment:
The only "issue" I find is that both of these are snake_cased which differs from the camelCase we've used for everything else.